src/snowflake/cli/plugins/object/stage/diff.py

```diff
@@ -154,12 +161,33 @@ def build_md5_map(list_stage_cursor: SnowflakeCursor) -> Dict[str, str]:
     }
 
 
-def stage_diff(local_path: Path, stage_fqn: str) -> DiffResult:
+def get_absolute_files_to_stage(
+    relative_files_to_stage: List[Path], local_path: Path
+) -> List[Path]:
+    paths = []
+    for file in relative_files_to_stage:
+        absolute_path = Path(os.path.join(local_path, file))
```

Comment:
FYI, this is only an absolute path if `local_path` is an absolute path. I don't think they need to be.